mysql-test/main/mysqld--help.result

```diff
@@ -1861,6 +1865,7 @@ net-buffer-length 16384
 net-read-timeout 30
 net-retry-count 10
 net-write-timeout 60
+new-mode NOW_DEFAULT
```

Comment:
What's `NOW_DEFAULT` ?  I think the intent was that by default the variable would be empty.

Reply:
NOW_DEFAULT is a placeholder for features that were opt-in in previous versions, but are now default.
So for example, we introduce NEW_MODE_FIX_DISK_TMPTABLE_COSTS as a flag disabled by default.

>#define NEW_MODE_FIX_DISK_TMPTABLE_COSTS                               (1 << 1)

when we wish to make this default, all we need to do is..

>#define NEW_MODE_FIX_DISK_TMPTABLE_COSTS                               (1 << 0)

We wont _need to_ (though we should)  change the actual code itself.